bundle/config/environment.go

```diff
@@ -1,12 +1,22 @@
 package config
 
+type Mode string
+
+const (
+	Debug       Mode = "debug"
+	Default     Mode = "default"
+	PullRequest Mode = "pull-request"
+)
+
 // Environment defines overrides for a single environment.
 // This structure is recursively merged into the root configuration.
 type Environment struct {
 	// Default marks that this environment must be used if one isn't specified
 	// by the user (through environment variable or command line argument).
 	Default bool `json:"default,omitempty"`
 
+	Mode Mode `json:"mode,omitempty"`
```

Comment:
it should rather be a property of a bundle flipped by a command-line flag and not the environment variable.

https://github.com/databricks/cli/blob/ccbcccd92903bc0a6dbdbc94072035b30aca8527/cmd/bundle/destroy.go#L27C1-L28

Reply:
Thing is, it's not really "ephemeral" at this time. We need to create real, persistent jobs and pipelines. The main reason is that one job may need to reference another job or another pipeline in the same DAB.

When we say "ephemeral" in this case we really just mean that we need to have "some" garbage collection mechanism that removes these persistent jobs after perhaps 24 hours of non-use.

Reply:
we can add that cleanup capability for previous normal runs from bundle runner 🤷‍♂️